src/types/Story.ts

```diff
@@ -133,13 +177,7 @@ export namespace Story {
         CONFIDENTIAL = 'confidential',
     }
 
-    export interface ExtraFields {
-        /**
-         * Depending on `format_version` this field can contain:
-         * - HTML content for v1 stories (deprecated)
-         * - JSON-encoded structured content for v3 stories (see Prezly Content Format).
-         */
-        content: string;
+    export interface OnDemandFields {
```

Comment:
I can already see all the fun we'll have with updating SDK in theme-kit and on the website-related repos, considering this will be another major release :D

Reply:
But can you not just wait for the next important update and perform the necessary changes only when it's needed?

Reply:
My reasoning for this change is that `ExtendedStory` does not include all `ExtraFields`. Emphasizing the difference between `ExtendedStory` and on-demand includes felt like the right thing to do.

However, I can revert this change. If that will save a lot of time.

Reply:
1a9a038e5f56cd91777c7022e1c538b0166ef9ea